addons/ccm-digitalocean/ccm-digitalocean.yaml

```diff
@@ -14,6 +14,8 @@ spec:
     metadata:
       labels:
         app: digitalocean-cloud-controller-manager
+      annotations:
+        "credentials-hash": "{{ .CredentialsCCMHash }}"
```

Comment:
Should we add some prefixes to this annotation like "kubeone.k8c.io/"? I think making it explicit that this annotation is managed by KubeOne can be valuable for the end user.

Reply:
I added the `kubeone.k8c.io/` prefix for all KubeOne annotations.